frontend/src/components/Login/Login.tsx

```diff
@@ -294,7 +288,12 @@ export default function Login({ onLogin }: ILoginProps) {
                 </View>
                 <TouchableOpacity
                   onPress={
-                    isForgotPassword ? () => setIsForgotPassword(false) : chooseDifferentMail
+                    isForgotPassword
+                      ? () => {
+                          setIsForgotPassword(false);
+                          setShowConfirmButton(true);
+                        }
```

Comment:
put this in a function  if its more than one line.  or make a variable that defines which function to use